jobserver/management/commands/create_backend.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+
+from jobserver.models.backend import Backend
+from jobserver.models.user import User
+
+
+class Command(BaseCommand):
+    """
+    Create or update a backend
+
+    Designed to be used to automate development setup.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("slug", default="local-dev", help="The id for backend")
+        parser.add_argument(
+            "--name", default="Local Dev", help="The name will be set to this"
+        )
+        parser.add_argument(
+            "--url", default=None, help="level_4_url will be set to this"
+        )
+        parser.add_argument(
+            "--user", default=None, help="Username to add to the backend"
+        )
+
+    def handle(self, *args, **options):
+
+        backend, created = Backend.objects.get_or_create(slug=options["slug"])
+        backend.name = options["name"]
+        if options["url"]:
+            backend.level_4_url = options["url"]
+        backend.save()
+
+        if options["user"]:
+            user = User.objects.get(username=options["user"])
+            user.backends.add(backend)
+            user.save()
```

Comment:
This is entirely a matter of taste, would calling BackendMembership (the through model behind `user.backends`) be clearer to you here?

Reply:
yes, it would. idempotency iz gud.